src/runtime/manifest.md

```diff
@@ -174,35 +173,37 @@ Can include how particle inputs are connected to handles:
 ```
 ...
 recipe
-  map as handle1                 // maps a store external to the arc; changes in the
+  handle1: map                   // maps a store external to the arc; changes in the
                                  // external store will be reflected locally, but the
                                  // local handle is read-only.
-  use as handle2                 // uses some handle already in the Arc
-  create as handle3              // creates a new empty handle
-  copy ImportedHandle as handle4 // creates a handle on a new store populated with entries
+  handle2: use                   // uses some handle already in the Arc
```

Comment:
What happened to `use *` ? Did we abandon it? Or is it coming soon? Or is it optional?

Reply:
i don't think this is supported (yet?) @Cypher1

Reply:
Sorry, this was an oversight in my original changes.

For now its going to be optional (and not printed in toString by default).

Introduction of the star is here: https://github.com/PolymerLabs/arcs/pull/3996